README.md

```diff
@@ -32,7 +32,7 @@ See below for exact commands and available arguments.
 
 For full instructions:
 
-`epacems.py --help`
+`epacems --help`
```

Comment:
factory_boy is part of the test suite, strictly speaking it shouldn't harm anything except the tests if you don't load it.  I don't have access to your system but a quick web search on the error suggests you may need to install "wheel."

https://stackoverflow.com/questions/34819221/why-is-python-setup-py-saying-invalid-command-bdist-wheel-on-travis-ci

Odd that my system is resolving that dependency automatically and not yours, but it could be added to requirements if it turns out that the correct solution.

--

The scrapy error you showed does look like a network timeout;  you should be able to adjust it in the settings.py via a DOWNLOAD_TIMOUT setting: https://docs.scrapy.org/en/latest/topics/settings.html?highlight=DOWNLOAD_TIMEOUT#download-timeout

Scrapy also has a lot of flexibility with error handling and retries, depending on whether a timeout adjustment or handling failures is more appropriate to your network.

--

I'll follow up on the epacems script name during the day, assuming I can get my sleep cycle fixed.

Reply:
Okay great bumping the `DOWNLOAD_TIMEOUT` up fixed it.

Weird that `pip` isn't satisfying the `wheel` requirement. If it's only required for testing, would it make sense to add it as part of an `extras_require={"test": ["wheel",]}` argument in the call to `setup()` within `setup.py`?